packages/@uppy/core/src/Uppy.ts

```diff
@@ -2120,12 +2130,24 @@ export class Uppy<
         })
 
         const { fileIDs } = currentUpload
+        if (this.#uploaders.has(fn)) {
```

Comment:
I don't think we need this condition, we didn't  have it before
```suggestion
        {
```

Reply:
Fixed. I tend to be overly redundant at times.